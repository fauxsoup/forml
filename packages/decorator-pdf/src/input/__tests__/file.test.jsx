import File from '../file';
import React from 'react';
import { render } from '@testing-library/react';

describe('renders', function () {
    test('nothing', function () {
        const { container } = render(<File value={'/tmp/test'}>1</File>);
        expect(container).toMatchSnapshot();
    });
});
