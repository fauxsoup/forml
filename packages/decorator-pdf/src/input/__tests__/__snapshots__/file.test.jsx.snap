// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders nothing 1`] = `<div />`;
